include/RigidBody/Joints.h

```diff
@@ -163,6 +163,28 @@ class BIORBD_API Joints : public RigidBodyDynamics::Model
     int GetBodyRbdlIdToBiorbdId(
         const int idx) const;
 
+    ///
+    /// \brief Return the rbdl idx of child of each segments
+    /// \return the rbdl idx of child of each segments
+    ///
+    std::vector<std::vector<unsigned int> > GetChildRbdlIdx() const;
+
+    ///
+    /// \brief Return the rbdl idx of subtrees of each segments
+    /// \return the rbdl idx of subtrees of each segments
+    ///
+    std::vector<std::vector<unsigned int> > GetSubTrees();
+
+    ///
+    /// \brief Return the rbdl idx of subtrees of each segments
+    /// \param subTrees the rbdl idx of subtrees of each segments to be filled
+    /// \param idx starting index to explore the subtrees
+    /// \return the rbdl idx of subtrees of each segments starting from the specified index
+    ///
+    std::vector<std::vector<unsigned int> > recursiveSubTrees(
```

Comment:
Same comment
Also, if these functions are not supposed to be called by a lambda user (for instance, they don't return biorbd trees, they are returning rbdl subtrees, which would be incompatible with biorbd... and therefore may be confusing for a lambda user). Then mark them as `protected`

Reply:
protected function have to be at the end of the .h, right ?